### DISCUSSION TITLE
Parallel pre.backup.hook

### DISCUSSION THREAD

**Discussion by crdemon09** [vmware-tanzu/velero#5013] [Community support Q&A]
Hi team,
I want to ask if it's possible to perform a parallel pre.backup.hook.
For instance I have a mongodb and postgres and I want to perform lock on them in parallel - so they data will be consistent from backup side?
Kind Regards

**Comment by VphDreamer** [comment]
@crdemon09  Hello, I don't have an answer to your questions, but, It just happen that I saw this discussion while working also on mongodb backup, so, I'm trying to figure out what would happen if we don't lock the database before backup (like having a scenario, where we backup inconsistent data), but, It seems that i'm not able to generate similar scenarios, do you have some ideas about this ?
